client-js/app/elements/labrad-nodes.ts

```diff
@@ -63,8 +63,15 @@ export class LabradInstanceController extends polymer.Base {
     return places.serverUrl(instanceName);
   }
 
-  ready() {
-    this.updateButtonState(this.status);
+  attached() {
+    this.async(() => {
+      this.fire('labrad-instance-controller::ready', {
```

Comment:
Arbitrary strings, I did it to namespace things. We could choose any operator, I've just always used `::` in the past.